mlir/test/Dialect/Linalg/tile-sort.mlir

```diff
@@ -0,0 +1,88 @@
+// RUN: mlir-opt %s -transform-interpreter -split-input-file -debug-only=tile-using-interface 2>&1 | FileCheck %s
+
+func.func @tile_order_ceil_then_negf(%arg: tensor<256xf32>) -> tensor<256xf32> {
+  // Worklist:
+  // * Tiling linalg.powf -> {linalg.ceil (priority = 0), linalg.negf (priority = 1)}
+  // * Tiling linalg.ceil -> {linalg.negf (priority = 1)}
+  // * Tiling linalg.negf -> {linalg.ceil (priority = 0)}
```

Comment:
It's not clear to me how to read this. Same for the CHECKs below.
I expect something to say: Tiling powf first, then filing negf, then tiling ceil

Reply:
Thanks, that helps!